timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -250,15 +268,16 @@ def fit(
             values which are given in``autogluon/timeseries/trainer/models/presets.py``. Specific hyperparameter
             choices for each of the recommended models can be found in the references.
         hyperparameter_tune_kwargs : str or dict, default = None
-            # TODO
+            # TODO ?????
```

Comment:
Can wait until we overhaul HPO and Ray? But I'm OK to remove the TODO and just add a generic explanation with values it can take for now.

Reply:
How about we postpone the updated docstrings for `presets`, `hyperparameters` and `hyperparameter_tune_kwargs` to another PR (after we incorporate PyTorch models / TFT)?

Reply:
works for me 👍